cmd/snap/cmd_known.go

```diff
@@ -40,6 +42,7 @@ type cmdKnown struct {
 	} `positional-args:"true" required:"true"`
 
 	Remote bool `long:"remote"`
+	Direct bool `long:"direct"`
```

Comment:
This needs documentation (I mean a help string)

Reply:
Indeed, added that now.